pandas/tests/io/json/test_ujson.py

```diff
@@ -803,7 +826,16 @@ def test_0d_array(self):
         ],
     )
     def test_array_numpy_except(self, bad_input, exc_type, kwargs):
-        with pytest.raises(exc_type):
+        msg = (
+            "Cannot decode multidimensional arrays with variable length elements to "
+            "numpy|"
+            "argument must be a string, a bytes-like object or a number, not|"
+            "nesting not supported for object or variable length dtypes|"
+            r"invalid literal for int\(\) with base 10|"
+            "labels only supported up to 2 dimensions|"
+            "cannot reshape array of size"
```

Comment:
I don't think we need to add such a complex match, I would just do a "match all" here

Reply:
Ok, again I didn't want to do a match all, so I added the error message to the test parameters. It added a lot of lines due to the line length limit, but I think it's ok. Let me know what you think.

I wanted to try this, but if you really don't like it I will do it your way.